sdk/program/src/account_info.rs

```diff
@@ -394,6 +395,74 @@ impl<'a> AsRef<AccountInfo<'a>> for AccountInfo<'a> {
     }
 }
 
+// The VmSlice class as defined in bpf_loader
+#[derive(Clone)]
+#[repr(C)]
+pub struct VmSlice<T> {
```

Comment:
That's my "I don't know where to define VmSlice" issue. Hoping to have Anza tell me.. it can't really go in any of the files where it's used, because they're all in different places and at best I'd have to add "use MemoryMapping" to a place where it currently is not, which is a good indicator that I shouldn't put it there.  (You may recall last week I had a circular link error that was showing up.. it related to two files each pulling in the other due to this struct and the MemoryMapping one)

Reply:
Right, if it causes a circular dependency, then perhaps putting it in the crate that's a dependency of the other makes most sense. We can always go with what we think is best for now and move it if Anza disagrees.